API/Controllers/TournamentsController.cs

```diff
@@ -20,37 +22,27 @@ IAdminNoteService adminNoteService
     /// <summary>
     /// Get all tournaments which fit an optional request query
     /// </summary>
-    /// <param name="requestQuery">The optional request query filter</param>
     /// <remarks>Will not include match data</remarks>
     /// <response code="200">Returns all tournaments which fit the request query</response>
-    /// <returns>
-    /// A page of tournaments
-    /// </returns>
```

Comment:
Why remove these?

Reply:
When binding a complex object `[FromQuery]` adding a `<param>` tag does nothing, all of the documentation for the parameters live in the type itself. We shouldn't be using `<returns>` tags, as that is functionally what `<response code="200">` (or whatever successful code) is documenting